src/components/AdminPortal/EventRegistrantsModal/Modal/EventRegistrantsModal.tsx

```
@@ -38,94 +38,110 @@
  * - `NotificationToast` for toast notifications.
  * - `react-i18next` for translations.
  */
-import React, { useState, useEffect } from 'react';
-import Button from 'shared-components/Button';
+import React, { useEffect, useState } from 'react';
 import { useMutation, useQuery } from '@apollo/client';
+import Autocomplete from '@mui/material/Autocomplete';
```

ESLint

[error] '@mui/material/Autocomplete' import is restricted from being used. Do not import Autocomplete from `@mui/material`. Use the shared DropDownButton component with searchable={true} instead.

(no-restricted-imports)